src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(numbers) {
```

Comment:
The parameter name 'numbers' may be misleading since it implies multiple numbers could be passed, but the function is designed to take only a single number at a time. Consider renaming it to 'number' or another singular term.